src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -1,127 +1,147 @@
 <template id="limited-restoration-panel">
```

Comment:
ESLint did a log of cleanup here.

I did rename the radio button `name` and the text field `v-model`.